independent-publisher-connectors/OpenAI/OpenAI.swagger.json

```diff
@@ -0,0 +1,261 @@
+                  "default": 1
+                },
+                "temperature": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Higher values means the model will take more risks. Try 0.9 for more creative applications, and 0 (argmax sampling) for ones with a well-defined answer. Use this OR top p",
+                  "title": "temperature",
+                  "default": 1
+                },
+                "max_tokens": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "One token equals roughly 4 characters of text (up to 4000 tokens between prompt and completion)",
+                  "title": "max tokens",
+                  "default": 100
+                },
+                "top_p": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered.",
+                  "title": "top p"
+                },
+                "frequency_penalty": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Number between -2.0 and 2.0. Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the models likelihood to repeat the same line verbatim.",
+                  "title": "frequency penalty",
+                  "default": 0
+                },
+                "presence_penalty": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Number between -2.0 and 2.0. Positive values penalize new tokens based on whether they appear in the text so far, increasing the models likelihood to talk about new topics.",
+                  "title": "presence penalty",
+                  "default": 0
+                },
+                "user": {
+                  "type": "string",
+                  "title": "user",
+                  "description": "A unique identifier representing your end-user, which will help OpenAI to monitor and detect abuse"
+                },
+                "stop": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Up to 4 sequences where the API will stop generating further tokens. The returned text will not contain the stop sequence"
+                }
+              },
+              "required": [
+                "prompt"
+              ]
+            }
+          }
+        ],
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v1/engines": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@PowerRobin Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Deleted the whole "engines" action. Originally wanted to add it as a dynamic parameter, but it returns far too many options that are not viable and would confuse the user.